src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" 
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+          href="https://fonts.gstatic.com" 
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <img class="imac_photo"
+           src="images/imac.jpeg" 
+           alt="imac">
+      <div class="product">
+        <h1 class="product_name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <p class="product_code">Product code: 195434</p>
+      </div>
+      <div class="container__card card">
+        <div class="stars stars--4">
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <p class="product_reviews">Reviews: 5</p>
+        </div> 
+      </div>
+      <div class="product_price">
+        <span class="price">Price:
+        </span>$2,199
+      </div>
+      <a class="button"
+        href="#" 
+        data-qa="hover"
+      >BUY
+      </a>
```

Comment:
```suggestion
      <a 
        class="button"
        href="#" 
        data-qa="hover"
      >
        BUY
      </a>
```
check your code style